var/spack/repos/builtin/packages/apex/package.py

```diff
@@ -12,12 +12,13 @@ class Apex(CMakePackage):
     """Autonomic Performance Environment for eXascale (APEX)."""
 
     maintainers = ['khuck']
-    homepage = "https://github.com/khuck/xpress-apex"
-    url      = "https://github.com/khuck/xpress-apex/archive/v2.3.1.tar.gz"
-    git      = "https://github.com/khuck/xpress-apex"
+    homepage = "https://uo-oaciss.github.io/apex"
+    url      = "https://github.com/UO-OACISS/apex/archive/v2.3.1.tar.gz"
```

Comment:
Do the Spack maintainers care if the version on the "url" drifts from the most recent version?

Reply:
No, the URL is just relevant for the pattern. As long as all versions (including the old ones) are available there this should be fine.